interfaces/web-animations.idl

```diff
@@ -0,0 +1,177 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the Web Animations spec.
+// See https://w3c.github.io/web-animations/
```

Comment:
This should be https://drafts.csswg.org/web-animations/

Reply:
I'll defer that change for @foolip

Reply:
https://w3c.github.io/web-animations/ now redirects to https://drafts.csswg.org/web-animations/ but not using a HTTP redirect.

Sent two reviews to update URLs I found in Chromium, only the first actually relevant:
https://chromium-review.googlesource.com/c/chromium/src/+/970648
https://github.com/web-animations/web-animations-js/pull/191

@lukebjerring, have the other recent changes taken effect in the output of the IDL scraper? If so, then after the first has landed, another update should fix this.

Reply:
Ah, except the day-to-day has the wrong URL as well. Will fix.

Reply:
Updated the source (switched https://drafts.csswg.org/web-animations/ to be the canonical source and https://w3c.github.io/web-animations/ to be an `alt` source)